google_compute_engine_oslogin/authorized_keys/authorized_keys.cc

```diff
@@ -36,14 +36,17 @@ int main(int argc, char* argv[]) {
   }
   std::stringstream url;
   url << kMetadataServerUrl << "users?username=" << UrlEncode(argv[1]);
-  string user_response = HttpGet(url.str());
+  string user_response;
+  if (!HttpGet(url.str(), &user_response)) {
+    return 1;
+  }
   if (user_response.empty()) {
     // Return 0 if the user is not an oslogin user. If we returned a failure
```

Comment:
Yeah, I'd confirm that. If that's true, 200 with an empty body means something else then. Also, the 404 alone probably can't tell us whether it was because the user didn't exist, or the URL we used with curl was simply wrong. The latter is unlikely as it can be discovered and fixed in testing, so maybe we could assume 404 means "User Not Found". :)